packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -72,8 +77,8 @@ const Checkbox = React.forwardRef(
                         'dc-checkbox__box--disabled': disabled,
                         'dc-checkbox--grey-disabled': disabled && greyDisabled,
                     })}
-                    {...(withTabIndex?.length > 0 ? { tabIndex: withTabIndex } : {})}
-                    tabIndex='0'
+                    {...(withTabIndex && withTabIndex?.length > 0 ? { tabIndex: withTabIndex } : {})}
+                    tabIndex={0}
```

Comment:
```suggestion
                    tabIndex={withTabIndex || 0}
```
I believe extra check `withTabIndex &&` is redundant, and btw placing tabIndex={0} after destructuring the object with the same value leads to overriding tabIndex with 0 every time, right?
besides, if withTabIndex is just a string, why would we check its length instead of simply checking if its non-empty(truthy value)? 🙂

Reply:
@maryia-binary you're right. I didn't notice that. updated my code according to @farzin-fs and @likhith-deriv suggestions.